docs/source/api-reference/drivers/flashers.md

```
@@ -265,26 +269,26 @@ and metadata. It is a container that contains the following:
+| `manufacturer`       | Name of the device manufacturer                                            |         |
+| `link`               | URL to device documentation or manufacturer website                        |         |
+| `bootcmd`            | Command used to boot the device (e.g. booti, bootz)                        |         |
+| `default_target`     | Default target device to flash to if none specified                        |         |
+| `targets`            | Map of target names to device paths                                        |         |
+| `login.type`         | Type of login shell                                                        | busybox |
+| `login.login_prompt` | Expected login prompt string                                               | login:  |
+| `login.username`     | Username to log in with, leave empty if not needed                         |         |
+| `login.password`     | Password for login, leave empty if not needed                              |         |
+| `login.prompt`       | Shell prompt after successful login                                        | #       |
+| `preflash_commands`  | List of commands to run before flashing, useful to clear boot entries, etc |         |
+| `kernel.file`        | Path to kernel image within bundle                                         |
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 3; Actual: 2; Too few cells, row will be missing data

(MD056, table-column-count)